cdap-ui/app/directives/complex-schema/complex-schema.less

```diff
@@ -35,26 +36,37 @@ my-complex-schema {
 
     .input-column {
       float: left;
-      width: ~"-moz-calc(100% - 90px - 80px)";
-      width: ~"-webkit-calc(100% - 90px - 80px)";
-      width: ~"calc(100% - 90px - 80px)";
+      width: ~"-moz-calc(100% - 75px - 80px)";
```

Comment:
I think we could do `-moz-cal(100% - 155px)` but I think we want to convey the idea of why we are subtracting 155 (75+80) from 100% for the width. May be a small comment on the calculation?